packages/manager/modules/iam/src/components/entityName/entityName.component.js

```diff
@@ -0,0 +1,18 @@
+import controller from './entityName.controller';
+import template from './entityName.template.html';
+
+export default {
+  bindings: {
+    entity: '<?',
+    messages: '<?',
+    name: '@?',
+    ngModel: '<',
```

Comment:
**suggestion:** Better to use different name to avoid confusion between Angular's `ngModel`